entries/t/test.de.json

```
@@ -0,0 +1,17 @@
+{
+  "Stiftung Warentest": {
+    "domain": "test.de",
+    "url": "https://www.test.de/",
+    "tfa": [
+      "sms",
+      "email"
```

```suggestion
      "totp"
```

I don't see any evidence of SMS or email 2FA. From what I can see, only TOTP is supported.
If this is not true, please provide a screenshot or other proof of SMS and/or email 2FA.